modules/text_server_fb/gdextension_build/SConstruct

```diff
@@ -217,7 +217,7 @@ if env["platform"] == "macos":
     )
 else:
     library = env.SharedLibrary(
-        f'./bin/libtextserver_fallback.{env["platform"]}.{env["target"]}.{env["arch_suffix"]}{env["SHLIBSUFFIX"]}',
+        f'./bin/libtextserver_fallback.{env["platform"]}.{env["target"]}.{env["suffix"]}{env["SHLIBSUFFIX"]}',
```

Comment:
Reading https://github.com/godotengine/godot-cpp/pull/867, I think you're meant to replace the whole `env["platform"]}.{env["target"]}.{env["arch_suffix"]}` with `{env["suffix"]}`.

Reply:
Indeed, it seems to include it now.